### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by Josh-Cena**
Mostly wrong macros used.

### CODE REVIEW

File: files/en-us/web/api/xrinputsource/targetraymode/index.md [hunk 1]
```diff
@@ -79,4 +79,4 @@ See the article [Inputs and input sources](/en-US/docs/Web/API/WebXR_Device_API/
 
 - [WebXR Device API](/en-US/docs/Web/API/WebXR_Device_API)
 - [Inputs and input sources](/en-US/docs/Web/API/WebXR_Device_API/Inputs)
-- [Using gamepads in WebXR applications](/en-US/docs/Web/WebXR%20Device%20API/Gamepads)
+- [Using gamepads in WebXR applications](/en-US/docs/Web/API/WebXR_Device_API/Gamepads)
```
Comment (estelle) [inline files/en-us/web/api/xrinputsource/targetraymode/index.md:82]: this link doesn't exist (used a few times here), but there is  a`Web/API/Gamepad` directory

File: files/en-us/web/api/ext_disjoint_timer_query/index.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ WebGL extensions are available using the {{domxref("WebGLRenderingContext.getExt
 
 > **Note:** This extension should be available in {{domxref("WebGLRenderingContext", "WebGL1", "", 1)}} contexts only. {{domxref("EXT_disjoint_timer_query_webgl2")}} is available in {{domxref("WebGL2RenderingContext", "WebGL 2", "", 1)}} contexts .
 >
-> In WebGL 2, the {{domxref("getQueryObject")}} was renamed to {{domxref("getQueryParameter")}}.
+> In WebGL 2, the {{domxref("WebGLRenderingContext.getQueryObject")}} was renamed to {{domxref("WebGL2RenderingContext.getQueryParameter")}}.
```
Comment (estelle) [inline files/en-us/web/api/ext_disjoint_timer_query/index.md:17]: the first link doesn't seem to exist (though I do see it under the current EXT directory

File: files/en-us/web/api/dommatrix/dommatrix/index.md [hunk 3]
```diff
@@ -33,7 +33,7 @@ new DOMMatrix(init)
 ## Examples
 
 This example creates a DOMMatrix to use as an argument for calling
-{{domxref("Point.matrixTransform()")}}.
+{{domxref("DOMPointReadOnly.matrixTransform()")}}.
```
Comment (estelle) [inline files/en-us/web/api/dommatrix/dommatrix/index.md:36]: Links to non-existant file

**Review by estelle — COMMENTED** [review]
not sure if linking to missing files was intentional or not.

**Comment by Josh-Cena** [comment]
The new link is intentionally still broken. There are other places referencing this link and I would assume it's just an unwritten documentation since our WebXR docs are incomplete.

**Review by estelle — APPROVED** [review]

**Pull request merged**